packages/sei-cosmwasm/README.md

```diff
@@ -38,6 +38,16 @@ Currently, Sei Bindings support query and message support for the sei custom mod
     - Query
         - Epoch
             - Get current epoch information
+- TokenFactory
+    - Message
+        - CreateDenom
+            - Creates a denom of `factory/{creator address}/{subdenom}` given the denom creator address and the subdenom. Subdenoms can contain `[a-zA-Z0-9./]`.
+        - MintTokens
+            - Mint an amount of denom. Minting of a specific denom is only allowed for the creator of the denom registered during `CreateDenom`.
+        - BurnTokens
+            - Burn an amount of a denom. Burning of a specific denom is only allowed for the creator of the denom registered during `CreateDenom`.
+        - ChangeAdmin
+            - Burning of a specific denom is only allowed for the creator of the denom registered during `CreateDenom`.
```

Comment:
nit: then needs to be updated to not just be copy paste

Reply:
Updated